frontend/src/data/tasks.json

```diff
@@ -194,7 +194,7 @@
       {
         "answerKey": "yes-kids",
         "descriptionEn": "If you meet the eligibility criteria for the child-rearing provision, you can apply for it as part of applying for CPP or separately. You will need to have the following information:\n- Name(s), date(s) of birth, and social insurance number(s) for each child you were rearing\n- Date of child's entry to Canada, if they were born outside Canada. You must also provide documentation to show that child's date of entry into Canada\n- Whether you or your spouse collected the Family Allowance or were eligible for the Canada Child Benefit\n- Periods of time when you were not the primary caregiver for each child",
-        "descriptionFr": "Si vous remplissez les critères d'admissibilité à la clause pour élever des enfants, vous pouvez la demander en même temps que votre demande de RPC ou séparément. Vous aurez besoin des informations suivantes\u00a0:\n- Nom(s), date(s) de naissance et numéro(s) d'assurance sociale de chaque enfant que vous éleviez\n- Date d'entrée de l'enfant au Canada, s'il est né à l'étranger. Vous devez également fournir des documents attestant de la date d'entrée de l'enfant au Canada.\n- Si vous ou votre conjoint avez bénéficié de l'allocation familiale ou avez eu droit à l'allocation canadienne pour enfants\n- Périodes pendant lesquelles vous n'avez pas été le principal responsable de chaque enfant",
+        "descriptionFr": "Si vous remplissez les critères d'admissibilité à la clause pour élever des enfants, vous pouvez la demander en même temps que votre demande de RPC ou séparément. Vous aurez besoin des informations suivantes\u00a0:\n- Nom(s), date(s) de naissance et numéro(s) d'assurance sociale de chaque enfant que vous éleviez;\n- Date d'entrée de l'enfant au Canada, s'il est né à l'étranger. Vous devez également fournir des documents attestant de la date d'entrée de l'enfant au Canada;\n- Si vous ou votre conjoint avez bénéficié de l'allocation familiale ou avez eu droit à l'allocation canadienne pour enfants;\n- Périodes pendant lesquelles vous n'avez pas été le principal responsable de chaque enfant.",
```

Comment:
The list items in this block have semicolons at the end, and a period to finish. Some of the other unordered lists in the application don't follow this same format. Was this intentional?